client/src/main/webjar/program-details/program-details.module.js

```diff
@@ -0,0 +1,16 @@
+(function (angular) {
+    "use strict";
+
+    angular.module(
+        "healthBam.programDetails",
+        [
+            "ngMaterial",
+            "ui.router",
+            "healthBam.templates",
+            "healthBam.main",
+            "healthBam.program",
+            "healthBam.errorHandling"
+        ]
+    );
```

Comment:
Creates the module for the new component, and declares all its module dependencies.